schema/Retail/RET11/on_search.ts

```diff
@@ -339,6 +338,14 @@ export const FnBonSearchSchema = {
                                   },
                                 },
                               },
+                              allOf: [
+                                {
+                                  if: { properties: { holidays: { minItems: 1 } } },
```

Comment:
there should be no condition on holidays property. It's mandatory property; empty array is also allowed.